run-custom-mao.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -xe
+
+# This script needs an machine-api-operator image as an input parameter
+# For example: ./run-custom-mao.sh quay.io/sdasu/machine-api-operator:sdasu-fix
+export SCRIPTS_DIR=$(pwd)
+export CUSTOM_MAO_BUILD=$1
+
+kubectl get secrets --namespace openshift-machine-api | grep mariadb
+
+# Stop CVO, MAO, machine-api-controller and Metal3 deployments
+oc --config /home/stack/dev-scripts/ocp/auth/kubeconfig scale deployment --replicas=0 cluster-version-operator --namespace openshift-cluster-version
```

Comment:
this config path will not always be correct

Reply:
True. Let me fix that.